receiver/oracledbreceiver/metadata.yaml

```diff
@@ -387,3 +387,13 @@ metrics:
       value_type: int
       input_type: string
     unit: "{executions}"
+  oracledb.logons:
```

Comment:
If we ever possible can have `oracledb.logon.<something else>` metric, this should be `oracledb.logon.count`. If not, this name is good.

See https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#naming-rules-for-counters-and-updowncounters for more guidelines

Reply:
We might add the following metrics later (based on `v$statname`):
- `oracledb.user_logons`: only user logons operations (subset of `oracledb.logons` which counts also the background processes)
- `oracledb.current_logons`: current number of logons to the DB

Both are count metrics. I can't imagine any non-count metrics in this namespaces. But let me know if you nevertheless prefer a separate namespace with `count`, e.g. `oracledb.logon.count`.

```
SQL> select name from v$statname where name like '%logon%';

NAME
----------------------------------------------------------------
logons cumulative
logons current
user logons cumulative
```